example/k3d/scripts/smoke-test.bash

```diff
@@ -85,18 +84,13 @@ run() {
     k3d image import -c $K3D_CLUSTER_NAME \
       grafana/agent:main \
       grafana/agentctl:main \
-      grafana/agent-crow:main
+      us.gcr.io/kubernetes-dev/grafana/agent-crow:dev.main \
+      us.gcr.io/kubernetes-dev/grafana/agent-smoke:dev.main
```

Comment:
```suggestion
      us.gcr.io/kubernetes-dev/grafana/agent-crow:main \
      us.gcr.io/kubernetes-dev/grafana/agent-smoke:main
```

Reply:
@rfratto one more... the one that got merged had the wrong tag (dev.main vs main)